### ISSUE TITLE
IBC Steering Committee Meeting 2020

### ISSUE THREAD

**Issue by cwgoes** [cosmos/ibc#306]
To be scheduled & coordinated. Ideally in person.

**Comment by Michelle-L** [comment]
Something to add would a possible application process for the IBC working or steering committee?

**Comment by cwgoes** [comment]
May be Q2, this should be post-1.0-launch.

**Comment by Michelle-L** [comment]
@cwgoes is there a timeframe or date we should schedule for the IBC steering Committee invitations? Would April be an option?

**Comment by cwgoes** [comment]
I don't think we can schedule this until 1.0 launch, it is just too uncertain.

**Comment by cwgoes** [comment]
Physical meetings are not possible at the moment; closing for now.

**Comment by gjermundgaraba** [comment]
This issue was mistakenly reopened during some integration work, so closing again now.
